proto/collection.proto

```diff
@@ -117,43 +129,63 @@ message Card {
         CARD_TYPE_PODCAST = 1;
         CARD_TYPE_VIDEO = 2;
         CARD_TYPE_CROSSWORD = 3;
-        CARD_TYPE_DISPLAY = 4;
+        CARD_TYPE_DISPLAY = 4; // Display cards have their own separate design (see figma designs for ref).
         CARD_TYPE_MOSTVIEWED = 5;
     }
     CardType type = 1;
     Article article = 2;
-    optional bool boosted = 3;
-    optional bool compact = 4;
+    optional bool boosted = 3; // Boosted cards show a boosted headline size and a main image that spans full width on mobile.
+    optional bool compact = 4; // Compact cards don't show all the information that non-compact cards do, and tend to appear in a carousel.
     repeated Article sublinks = 5;
     optional string html_fallback = 6;
+    /**
+     * Individual cards can be branded (and not be part of a branded container). Cards that are branded tend to show the
+     * sponsor logo and should be returned with a different palette.
+     */
     optional Branding branding = 7;
+    /**
+     * Individual cards can be defined as "premium content". If premium_content is true then it implies the card should be
+     * hidden from signed-in users (for example if the card has been paid for by an external sponsor).
+     */
     optional bool premium_content = 8;
     optional Palette sublinks_palette_light = 9;
     optional Palette sublinks_palette_dark = 10;
 }
 
 message Column {
+    /**
+     * By default, if there are multiple cards in the cards array then it's expected that
+     * the client will display these as stacked vertical elements.
+     */
     repeated Card cards = 1;
     optional Palette palette_light = 2;
     optional Palette palette_dark = 3;
-    int32 preferred_width = 4;
+    int32 preferred_width = 4; // The card(s) in the column assume this preferred width.
```

Comment:
This is an valuable clarification, it's good to ensure that everyone has the same understanding here - for myself, when I saw `preferred_width` on the `Column` object, I assumed it meant the width of the _column_ rather than of the cards _in_ the column.

For myself, I feel like `preferred_card_width` would have been a better name for this field, but in any case, great work documenting it!